### PR TITLE
Better manage permissions modal

### PR DESCRIPTION
**Pull request by kocsmy**
I've improved the modal with the help of @kravets-levko 

- Dropdown now has proper spacing around its elements
- Query owner is no longer in the dropdown options (and can't be added either)
- Query owner is presented in the first line of the table by default

image

### CODE REVIEW

File: client/app/pages/queries/view.js [hunk 1]
```diff
@@ -487,6 +487,7 @@ function QueryViewCtrl(
       component: 'permissionsEditor',
       resolve: {
         aclUrl: { url: `api/queries/${$routeParams.queryId}/acl` },
+        query: $scope.query,
```
Comment (arikfr) [inline client/app/pages/queries/view.js]: This same dialog is used for dashboards too. It makes more sense to pass in the id of the user who owns the object.

File: client/app/assets/less/redash/query.less [hunk 2]
```diff
@@ -538,6 +538,16 @@ nav .rg-bottom {
   padding: 0 0 0 23px;
 }
 
+.table--permission {
+  .profile__image {
+    margin-right: 0;
+  }
+}
+
+.mp__permission-type {
```
Comment (arikfr) [inline client/app/assets/less/redash/query.less]: What's `mp`? And why two styles of naming? (`--` vs `__`)

Comment (kocsmy) [reply client/app/assets/less/redash/query.less]: mp refers to manage permission but ended up not using elsewhere, maybe the table should be mp__table but I chose to keep it aligned with the other table class names: "table table-condensed table-hover table--permission".

That's not 2 styles, it's BEM: http://getbem.com/naming/

File: client/app/pages/queries/view.js (hunk 1 above)
Comment (kocsmy) [reply client/app/pages/queries/view.js]: Oh it's being used in dashboards, too! @kravets-levko can you help me fix it so it works on dashboards, too?

Comment (kravets-levko) [reply client/app/pages/queries/view.js]: @arikfr (cc @kocsmy) Fixed

**Comment by kocsmy** [comment]
I checked it and all looks good 👌 Thanks, Levko!

**Comment by arikfr** [comment]
👍

**Pull request merged**